services/essentials.yaml

```diff
@@ -62,7 +62,7 @@ spec:
       version: '1.17'
     - groupId: org.jenkins-ci.plugins
       artifactId: jackson2-api
-      version: 2.9.7.0-beta
+      version: 2.9.7.1
```

Comment:
others are in `'`, but not this one. expected?

Reply:
shitty version parser and JSON FTW. Will be parsed as numeric or string depending mostly randomly :). Also it tries generally to compare using semver rules :man_facepalming:.
TL;DR: We need to replace that component, but it's not critical for now.

Reply:
YAML will parse floats as floats basically. So version `1.10` would be parsed as `1.1`, whereas version `'1.10'` will be parsed as the string `1.10`. Versions with at least two dots won't be parsed as numbers, though, hence the shorthand here.

Reply:
Yeah, I tried it. But basically, once you add things under `spec`, then using `make -C services generate-essentials` will compute what's needed under `status` and rewrite the whole file removing some of these. Again I didn't take time to dig into that either.

Reply:
YAML parsers will usually print out the YAML in the simplest way it can, so it'll only quote version strings that are ambiguous. There's probably a formatting option in whichever YAML library you're using here.